FALSEHOODS.md

```diff
@@ -135,9 +135,59 @@ surprising how many false assumptions programmers continue to make about them.
 
 1.  **Phone numbers contain only digits**
 
-    In Israel, certain advertising numbers start with a `*`.
+    In Israel, certain advertising numbers start with a `*`. In New Zealand,
+    non-urgent traffic incidents can be reported by calling `*555` from a mobile
+    phone.
 
 1.  **Phone numbers are always written in ASCII**
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  **Phone numbers have only one prefix (area code or national destination code) at a given time**
+
+    In the mid-90s in Iceland, phone numbers changed from 5 and 6 digits to 7 digits.
+    During the transition period, phone numbers could be reached by the old area code
+    or the new 7 digit number (a different prefix).
```

Comment:
I think we need a specific example here to make it clear that it wasn't just a new digit in front (comments on that PR).